react/features/invite/components/add-people-dialog/native/InviteButton.js

```diff
@@ -36,7 +42,13 @@ class InviteButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        this.props.dispatch(setAddPeopleDialogVisible(true));
+        const { _addPeopleEnabled, dispatch } = this.props;
```

Comment:
This looks the same code as my lonely experience share button code. Don't we want to extract it to a function?